src/platforms/flutter/migration.mdx

```diff
@@ -3,6 +3,51 @@ title: Migration Guide
 sidebar_order: 1000
 ---
 
+## Migrating from `sentry_flutter` `5.0.0` to `sentry_flutter` `5.1.0`
+
+# Android specific changes
+
+You may remove `android:extractNativeLibs="true"` meta-data in the `AndroidManifest` file or `android.bundle.enableUncompressedNativeLibs=false` in the `gradle.properties` file if you're using the [Android Native Development Kit](/platforms/android/using-ndk/). Sentry can now symbolicate events with the default value of [extractNativeLibs](https://developer.android.com/studio/releases/gradle-plugin#extractNativeLibs) and [android.bundle.enableUncompressedNativeLibs](https://developer.android.com/studio/releases/gradle-plugin#behavior-changes).
+
+Further instructions can be found [here](/platforms/android/migration/).
+
+User identification based on the [Android ID](https://developer.android.com/reference/android/provider/Settings.Secure#ANDROID_ID),
+which was the default behavior of Sentry Android 4, is now deprecated.
```

Comment:
I believe the text written on Android changelog is a bit more complete, can we copy and paste it?

https://docs.sentry.io/platforms/android/migration/

> A random generated installationId replaces Settings.Secure.ANDROID_ID, which has been removed. This may affect the number of unique users displayed on the the Issues page and Alerts. If you always set a custom user using Sentry.setUser(customUser), the behavior has changed. While you don't have to make any update, if you want to maintain the old behavior, use the following code snippet:

we'd just need to adapt in order to account for `device_info_plus`